### DISCUSSION TITLE
PAGO por mudanças no sistema.

### DISCUSSION THREAD

**Discussion by SuperPorte** [RamonSilva20/mapos#1814] [General]
Saudações pessoal!!!

Estou precisando fazer algumas mudanças no MAP-OS para atender meu negócio mas não sei nada de programação.
Como funciona para eu pedir para mudar e pagar pelas mudanças?

Eu até disponibilizaria depois no GIT, para que as atualizações futuras já viessem prontas, mas não tenho ideia de por onde começar para fazer isso.

Alguém me orienta nisso? Obrigado.

**Comment by SuperPorte** [comment]
Ninguém?????

**Comment by SuperPorte** [comment]
Pessoal, algum indica algum programador pra essas mudanças.
Deu tão certo quando fizeram a campanha anterior....

**Reply by jsfachini** [reply]
@luizrn

**Reply by SuperPorte** [reply]
Tentei mandar email.pra ele mas retornou assim:

--- 
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of
its recipients. This is a permanent error.

The following address failed:

    luizfernandorivanekel@hotmail.com:
        SMTP error from remote server for MAIL FROM command, host: hotmail-com.olc.protection.outlook.com (104.47.4.33) reason: 550 5.7.1 Service unavailable, Client host [82.165.159.131] blocked using
Spamhaus. To request removal from this list see https://www.spamhaus.o
rg/query/ip/82.165.159.131 (AS3130). [AM5EUR02FT092.eop-EUR02.prod.pro
tection.outlook.com]
---

E agora? (TºT)

**Reply by wnjulio** [reply]
31 992233670 chama esse ai

**Reply by SuperPorte** [reply]
@wnjulio To tentando contato com esse numero mas zero resposta.

ToT

**Reply by mrsousa01** [reply]
oi

**Comment by SuperPorte** [comment]
Alterações que preciso:
1- margem de lucro, no cadastro de produtos, fique salva no cadastro. Assim, ao comprar, vou informar a quantidade e preços novos e o sistema vai alterar o preço de venda automático;
2- Abater do estoque SOMENTE DEPOIS da OS APROVADA. Hoje o sistema abate do estoque logo ao lançar na OS. Mas se ainda estamos na fase de orçamento e negociação, não tem porque abater estoque;
3- Criar uma gestão de etapas da OS. Eu queria reorganizar os nomes dos STATUS das OSs. Mas mesmo que eu o faça, ao atualizar, eu perco as mudanças;
4- Filtrar a fila por técnico;
5- Termo de garantia com variáveis. Igual com as notificações. Poder trazer o nome do cliente, produtos/serviços, datas etc e tal. Automaticamente para CADA OS e VENDA;
6- Criar um recibo de OS. Hoje o sistema imprime um via de orçamento. MUITO boa até. Mas queria algo como um canhoto que eu destacasse e entregasse ao cliente;
7- Desconto em valor e não apenas em porcentagem;
8- Melhorar o controle de equipamentos do cliente/OS (cadastrando marca, modelo, SN,etc.);
9- Ma tela de vendas, poder vender serviços também. Hoje o sistema é limitado a vender produtos.

10- Cliente aprovar  ou rejeitar OS na "Área do cliente";
11- Lançar pagamentos DENTRO da OS para imprimir para o cliente;
12- Impressão de recibo dos LANÇAMENTOS financeiros (Ex: R$12 comprar garrafão de água 20lt);
13- Que o envio da OS por whatsapp saia exatamente como escrito na OS. Hoje o sistema pega o texto escrito e remove toda a formatação (quebra de linhas, enumeração, etc.) do texto e envia um bloco de informações que, pra maioria dos cliente, fica confuso.

14- Que a OS tenha, ALÉM de técnico responsável, um vendedor.

Que eu me lembre é isso. Se eu lembra de mais alguma coisa, adiciono aqui.
Valeuuu

**Reply by mrsousa01** [reply]
já fizeram o serviço ?

**Reply by SuperPorte** [reply]
Opa!! Bom dia!!
ainda não.

**Reply by mrsousa01** [reply]
Oi bom dia, vamos implementgar esta a função... vamos conversar no privado...

**Reply by mrsousa01** [reply]
qual seu zap ?

**Reply by SuperPorte** [reply]
Opa!!!
Segura aí: 27 992882865

Valeuuuu

**Reply by KelvinEger** [reply]
@bernarcom se com o @mrsousa01 não rolar, me chama que a gente desenvolve!
kelvin.eger@hotmail.com

**Reply by j7info** [reply]
Seria muito bom, se no relatório aparecesse o preço de compra e o preço de venda dos produtos.

**Reply by j7info** [reply]
Uma pergunta. Essas mudanças, vão para a versão que todos usam? Ou ficam apenas com quem pagou por elas? Tenho interesse em fazer algumas implementações, conforme minha necessidade aqui, mas também não sou programador.

**Reply by marques823** [reply]
O meu tem quase todas essas alterações, mas eu investir mais tempo em um módulo de proposta comercial, tem melhorias em controle de estoque, margem de lucro, filtros, fluxo de caixa, cadastros de produtos e clientes, e pode rodar facilmente em Docker Container que permite uso remoto, está disponível no Github nessa conta aqui.

**Comment by KelvinEger** [comment]
Bom dia J7Info, tudo beleza?
O outro jovem que solicitou as alterações não falou comigo ainda.

Eu posso realizar estas alterações pra você, caso desejas. Caso você pague,
você quem decide se as alterações vão para o projeto principal no github,
ou somente ficam com você.

Em qui., 16 de jun. de 2022 às 23:21, j7info ***@***.***>
escreveu:

**Reply by j7info** [reply]
Opa. Tenho interesse sim. Como falamos para combinar? Vou levantar os pontos desejados e vc vê quanto vai ficar. Se é pra ajudar, havendo ceitação, o que fizer por minha conta, pode sim, ser implantado para todos.

**Reply by KelvinEger** [reply]
Bom dia J7Info, respondi por e-mail mas parece que a resposta não veio parar aqui rs. 
Monte a lista do que deseja, e me mande um e-mail ou whats com a lista, que conversamos direto por lá. 
Meu e-mail está no meu perfil.

**Comment by SuperPorte** [comment]
Mil perdões pessoal. meus últimos dias foram MUITO intensos.
sextas sábados e domingos eu estou em um cliente de contrato e não abro o whatsapp.
eu quero abrir o financiamento igual foi há um tempo atrás. e disponibilizar a todos as mudanças. mas ainda nenhum progresso que contactar alguém pra fazer ToT

**Comment by SuperPorte** [comment]
Apesar de ausente a idéia ainda está de pé ok???

**Comment by KelvinEger** [comment]
Boa tarde.
Preciso saber somente o que vocês querem de alterações.

Em qua., 19 de out. de 2022 às 11:39, bernarcom ***@***.***>
escreveu:

**Reply by SuperPorte** [reply]
De minha parte é tudo isso:

https://github.com/RamonSilva20/mapos/discussions/1814#discussioncomment-2849116

**Comment by KelvinEger** [comment]


**Reply by SuperPorte** [reply]
lança o desafio do PIX que eu participo

**Comment by gilmariosantos** [comment]
Essas alterações evoluíram?

**Comment by AcessoHAG** [comment]


**Reply by marques823** [reply]
Olá, o que você automatizou com n8n em seu sistema? Pretende disponibilizar algo no git?

**Reply by tiagobeulke** [reply]
@AcessoHAG tenho interesse nas mudanças que já fez e também em UM PDV para frente de caixa

**Comment by maycon1977** [comment]
Maneiro, hein?
Como funcionaria essa integração com I.A. e N8N?

No que facilita o uso do Map-Os?

*Atenciosamente,*
*Maycon Martins Teixeira ***@***.***_rio)*

*Telefone/**WhatsApp**: +55 (
<https://api.whatsapp.com/send?phone=5521988078360&text=Ol%C3%A1%2C%20Maycon.%20Sou%20um%20cliente%20da%20Press%C3%A3oRio.%20Pode%20me%20ajudar%3F>**21)
98807-8360
<https://api.whatsapp.com/send?phone=5521988078360&text=Olá%2C%20Maycon.%20Sou%20um%20cliente%20da%20PressãoRio.%20Pode%20me%20ajudar%3F>*


Em qua., 3 de dez. de 2025 às 12:22, HEITOR ARAUJO GOMES <
***@***.***> escreveu:

**Comment by andremp15** [comment]
Essas alterações que você quer amigo não compensa fazer no sistema mapos, praticamente você tá pedindo a alteração do sistema inteiro é mais fácil criar um novo sistema para você se interessar é só entrar em contato comigo. Wh- 11958284889
